profiles/ros/python

```diff
@@ -11,4 +11,28 @@
 # Include python abstractions and excutables needed for ros with python
 #include <abstractions/python>
 
-/usr/bin/python{2,2.[4-7],3,3.[0-5]} rix,
+# CHECKME: Which are general for other languages?
+/bin/dash mrix,
+/bin/uname mrix,
+
+# CHECKME: Is this already in abstractions somewhere?
+@{PROC}/@{pid}/cgroup r,
+@{PROC}/@{pid}/cmdline r,
+@{PROC}/@{pid}/environ r,
+@{PROC}/@{pid}/fd/ r,
+
+# CHECKME: is `python3.[0-6]` too relaxed?
```

Comment:
Python 3.5 (xenial uses 3.5.1) and higher should be enough

Reply:
Ok, I'll ratchet that up to  `python3.[5-6]` then.